css-layout-classes/css-layout-classes-quiz-notes.md

```diff
@@ -0,0 +1,42 @@
+# css-layout-classes-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Why do two div elements "vertically stack" on one another by default?
+Becausse div elements are block elements.
+- What is the default `flex-direction` of an element with `display: flex`?
+Row
+- What are the three primary components of a page layout? (Which helper classes do you need?)
+Div, Article, Img
```

Comment:
Not quite! Think about how a page is broken up. Think of horizontal & vertical space. Look at the class names you were using for this exercise